node/node_syncing.go

```diff
@@ -338,7 +342,18 @@ func (node *Node) StartGRPCSyncClient() {
 			Msg("SupportBeaconSyncing")
 		go node.doBeaconSyncing()
 	}
-	node.supportSyncing()
+
+	if node.HarmonyConfig.General.UseTiKV {
+		node.syncFromTiKVWriter()
+
+		if node.HarmonyConfig.TiKV.Role == "Writer" {
+			node.supportSyncing()
+		} else {
+			node.Consensus.UpdateConsensusInformation()
+		}
+	} else {
+		node.supportSyncing()
+	}
```

Comment:
why this is initialised in the `StartGRPCSyncClient` method? does it make sense to move this block to it's own method and call it independently? 

second question for learning purposes, whats the `UpdateConsensusInformation` for?

Reply:
This was addressed by me with the offline commit